roxctl/generate/netpol/cta.go

```diff
@@ -54,8 +84,7 @@ func (cmd *generateNetpolCommand) generateNetpol() error {
 		return nil
 	}
 
-	cmd.env.Logger().WarnfLn("No mode selected, not providing any output.")
-	return nil
+	return errors.New("no mode selected, not providing any output")
```

Comment:
Is there any test for this change?

Reply:
This corner is actually impossible to reach. I removed this error and made output to stdout a default behavior.

Reply:
Solved with  [e23f3e5](https://github.com/stackrox/stackrox/pull/2637/commits/e23f3e58b451eafda654c03e4f1f71aa7365f07b)